portality/static/js/doaj.fieldrender.edges.js

```diff
@@ -2065,8 +2065,18 @@ $.extend(true, doaj, {
 
             this.namespace = "doaj-public-search";
 
+            this.selector = edges.getParam(params.selector, null)
+            this.currentQueryString  = "";
+
+
             this.draw = function () {
-                var frag = "<li class='alert'><p><strong>No results found</strong> that match your search.</p> <p>Try removing some of the filters you have set, modifying the text in the search box, or using less specific search terms.</p></li>";
+                if (this.component.edge.currentQuery){
+                    this.currentQueryString = this.component.edge.currentQuery.queryString.queryString;
```

Comment:
This looks error prone to me.  I'd try something like

```
if (this.component.edge.currentQuery) {
    let qs = this.component.edge.currentQuery.getQueryString();
    if (qs) {
        this.currentQueryString = qs.queryString || "";
    }
}
```

Reply:
done.